scylla-cql/src/frame/request/query.rs

```diff
@@ -204,3 +206,76 @@ impl<'q> QueryParameters<'q> {
         })
     }
 }
+
+#[derive(Debug, Clone)]
+pub enum PagingStateResponse {
+    HasMorePages { next_page_handle: PagingState },
+    NoMorePages,
+}
```

Comment:
Why 'handle'?

Reply:
Because it can be used to obtain the next page, this way serving as its handle.

Reply:
For me a handle is some (usually small / simple) object that allows access to some (bigger / allocated, not sure what is the right word here) resource - like a file handle, or a PHANDLE in win32 APIs.
Why not `HasMorePages { state: PagingState },`?